vpd-manager/src/worker.cpp

```diff
@@ -1421,6 +1437,9 @@ types::VPDMapVariant Worker::parseVpdFile(const std::string& i_vpdFilePath)
 std::tuple<bool, std::string>
     Worker::parseAndPublishVPD(const std::string& i_vpdFilePath)
 {
+    const std::string& l_inventoryPath =
+        jsonUtility::getInventoryObjPathFromJson(m_parsedJson, i_vpdFilePath);
```

Comment:
This API can throw exceptions and this can go uncaught. This will break the threading for all other collections.

Let the variable be at outer scope, bring the API call to "getInventoryObjPathFromJson" under try/catch.

Reply:
sure